SingularityService/src/main/java/com/hubspot/singularity/mesos/SingularityStartup.java

```diff
@@ -123,7 +139,15 @@ void checkSchedulerForInconsistentState() {
         case ACTIVE:
         case SYSTEM_COOLDOWN:
         case DEPLOYING_TO_UNPAUSE:
-          checkActiveRequest(requestWithState, deployKeyToPendingTaskId, now);
+          if(checkActiveRequest(requestWithState, deployKeyToPendingTaskId, now)) {
+            numPendingStartupRequests++;
+            double startUpTaskPct = (numPendingStartupRequests * 100.0 / totalStartupRequests);
```

Comment:
edit, ah I see the difficulty in trying to come up with a descriptive name for this now that I try and think of one, just don't want it to get confused with actual running tasks at all since we are counting enqueued pending requests